.github/ISSUE_TEMPLATE/Bug_report.yml

```
@@ -0,0 +1,46 @@
+name: Bug Report
+description: "Report something that's broken."
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "Before submitting your report, [please ensure your Laravel version is still supported](https://laravel.com/docs/releases#support-policy)."
+  - type: input
+    id: laravel-version
+    attributes:
+      label: Laravel Version
+      description: Please provide the full Laravel version you're using.
+      placeholder: 10.4.1
+    validations:
+      required: true
+  - type: input
+    id: php-version
+    attributes:
+      label: PHP Version
+      description: Please provide the full PHP version you're using.
+      placeholder: 8.1.4
+    validations:
+      required: true
+  - type: input
+    id: database-version
+    attributes:
+      label: Database Driver & Version
+      description: If applicable, please provide the database driver and full version you're using.
+      placeholder: "mysql Ver 8.0.31 for macos13.0 on arm64 (Homebrew)"
+    validations:
+      required: false
+  - type: textarea
```

```suggestion
  - type: markdown
```

Using this means you can get the full editor toolbar.

---

So this doesn't do what you think it does. I also used this in the beginning because I thought it would render a markdown editor. Turns out the markdown type is just so you can add some text in markdown to your form in between fields. The textarea is actually a markdown editor. Very confusing heh